website/www/site/content/en/get-started/quickstart-go.md

```diff
@@ -51,22 +43,18 @@ required arguments described in the examples.
 For example, to run `wordcount`, run:
 
 {{< runner direct >}}
-$ go install github.com/apache/beam/sdks/v2/go/examples/wordcount
-$ wordcount --input <PATH_TO_INPUT_FILE> --output counts
+$ go run github.com/apache/beam/sdks/v2/go/examples/wordcount@latest --input "gs://apache-beam-samples/shakespeare/kinglear.txt" --output counts
+$ less counts
 {{< /runner >}}
 
 {{< runner dataflow >}}
-$ go install github.com/apache/beam/sdks/v2/go/examples/wordcount
-# As part of the initial setup, for non linux users - install package unix before run
-$ go get -u golang.org/x/sys/unix
-$ wordcount --input gs://dataflow-samples/shakespeare/kinglear.txt \
+$ go run github.com/apache/beam/sdks/v2/go/examples/wordcount@latest --input gs://dataflow-samples/shakespeare/kinglear.txt \
```

Comment:
Should we use the same path `gs://apache-beam-samples/shakespeare/kinglear.txt` as above for consistency?